README.md

```diff
@@ -80,10 +80,9 @@ To learn more about the platform, check out our [documentation](https://github.c
 
   ```sh
   git clone https://github.com/ostis-ai/ostis-web-platform
-  cd ostis-web-platform
+  cd ostis-web-platform/scripts
   git checkout feature/component_manager
-  cd scripts
-  ./prepare.sh
+  ./install_platform.sh
```

Comment:
I run according this instructions and got 
![image](https://github.com/ostis-ai/ostis-web-platform/assets/8680298/9e2805c9-0a35-4678-a8a0-ce07cfef0b11)

Need to add steps about sc-machine installation. Or install_platfrom script should install sc-machine with sc-component-manager enabling. The last option is better in my pov